src/components/AllBeers.js

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { NavLink } from "react-router-dom";
+import Header from "./Header";
+
+function AllBeers() {
+  const [beers, setBeers] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://ih-beers-api2.herokuapp.com/beers")
```

Comment:
- Firstly, make sure you add you .env file to gitignore!! The are your private environment variables.
- Cool you created it, but still not using ;) 

Reminder of syntax:
```
 axios.get(`${process.env.REACT_APP_API_URL}/beers`)
```